java/ql/src/Security/CWE/CWE-295/ImproperWebViewCertificateValidation.ql

```diff
@@ -0,0 +1,18 @@
+/**
+ * @name Android `WebView` that accepts all certificates
+ * @description Trusting all certificates allows an attacker to perform a machine-in-the-middle attack.
```

Comment:
Sorry, commenting here to check that `machine-in-the-middle attack` is a thing (I've only ever heard of `man-in-the-middle` attacks 🙊 ).

Reply:
It should be man-in-the-middle, thanks

Reply:
I thought you used `machine-in-the-middle` as a gender-neutral term :-)

Reply:
In fact I copied it from another file. But that reasoning makes sense; I'll keep it as machine then